lib/node_modules/@stdlib/blas/base/srotg/docs/types/index.d.ts

```diff
@@ -0,0 +1,88 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2023 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 2.0
+
+/**
+ * Inteface describing `srotg`.
+ */
+interface Routine {
+    /**
```

Comment:
@Pranavchiku This all needs to be converted to TAB indentation.

Reply:
I guess I have provided TAB indentation. I checked it manually as well, I am not sure why it is throwing lint errors.